### PR TITLE
Create man-page for `inspect-container` command

### PR DESCRIPTION
**Pull request by keberle94**
Supersedes #1470

### CODE REVIEW

File: lib/hint.rb [hunk 1]
```diff
@@ -67,8 +67,7 @@ def do_complete_inspection(options)
          "#{program_name} inspect #{options[:host]} --name #{options[:name]} --extract-files"
       elsif options[:docker_container]
         "To do a full inspection containing all scopes and to extract files run:\n" \
-         "#{program_name} inspect-container --docker #{options[:docker_container]} " \
-         "--name #{options[:name]} --extract-files"
+         "#{program_name} inspect-container --name #{options[:name]} --extract-files"
```
Comment (aduffeck) [inline lib/hint.rb]: You can't remove the `#{options[:docker_container]}` here, it's necessary for the command to work.

File: lib/cli.rb [hunk 2]
```diff
@@ -618,21 +618,20 @@ def self.parse_inspect_command_options(host, options)
...
     Inspect container image and generate system descripton from inspected data.
+    Right now we only support docker images. Further types will be supported in the future as well.
 
     Multiple scopes can be passed as comma-separated list. If no specific scopes
     are given, all scopes are inspected.
 
     Available scopes: #{AVAILABLE_SCOPE_LIST}
   LONGDESC
-  arg "IMAGENAME"
+  arg "<IMAGENAME|IMAGEID>"
```
Comment (aduffeck) [inline lib/cli.rb]: Are the `<>` following some convention? We aren't using them in the other commands.

**Comment by aduffeck** [comment]
Can you please squash the commits that touch the man page into a single commit?

**Comment by keberle94** [comment]
Superseded by #1472

**Pull request closed without merging**